plugins/poll/assets/javascripts/discourse/components/poll-voters.gjs

```diff
@@ -0,0 +1,40 @@
+import Component from "@glimmer/component";
+import { fn } from "@ember/helper";
+import { on } from "@ember/modifier";
+import ConditionalLoadingSpinner from "discourse/components/conditional-loading-spinner";
+import DButton from "discourse/components/d-button";
+import avatar from "discourse/helpers/bound-avatar-template";
+import icon from "discourse-common/helpers/d-icon";
+import PollVotersIrv from "./poll-voters-irv";
+
+export default class PollVotersComponent extends Component {
+  get showMore() {
+    return this.args.voters.length < this.args.totalVotes;
+  }
+
+  <template>
+    <div class="poll-voters">
+      <ul class="poll-voters-list">
+        {{#if @isIrv}}
+          <PollVotersIrv @voters={{@voters}} />
+        {{else}}
+          {{#each @voters as |user|}}
+            <li>
+              {{avatar user.avatar_template "tiny"}}
+            </li>
+          {{/each}}
+        {{/if}}
+      </ul>
+      {{#if this.showMore}}
+        <ConditionalLoadingSpinner @condition={{@loading}}>
+          <DButton
+            {{on "click" (fn @fetchVoters @optionId)}}
+            class="poll-voters-toggle-expand"
+          >
+            {{icon "chevron-down"}}
+          </DButton>
```

Comment:
```suggestion
          <DButton
            @action=(fn @fetchVoters @optionId)
            @icon="chevron-down"
            class="poll-voters-toggle-expand"
          />
```

Reply:
hmm, prettier does not like `@action=(fn @fetchVoters @optionId)`

Reply:
changing it to `@action={{fn @fetchVoters @optionId}}`

Reply:
yes sorry :) doing it in github UI easy to do mistakes